### ISSUE TITLE
Flipping cards doesn't hide the card on hover.

### ISSUE THREAD

**Issue by zebington** [Cockatrice/Cockatrice#1790]
When you flip a card after it's ever been face up, it's P/T is set to the cards P/T.
![test](https://cloud.githubusercontent.com/assets/4019381/12804931/6256e448-caef-11e5-8486-df8a3d5f3794.png)

**Comment by skwerlman** [comment]
@Daenyth I think this issue is about more than P/T: From the image, you can still see that the face-down card is an ornithopter because the card image is still shown. Additionally, in chat it says "zebington untaps Ornithopter" _after_ it says "zebington flips Ornithopter face down".

**Comment by Nightfirecat** [comment]
@skwerlman Keep in mind this is from the POV of a player who owns the card. I believe in the case that you were on the other side of a match with flip cards, you should see "zebington untaps face-down card \#1".

**Comment by tooomm** [comment]
Yes, the other player shouldn't be able to see the former cards picture or name!

The "remove old P/T value on card flip down" seems to be a very helpful feature. :+1:

**Comment by ZeldaZach** [comment]
There's a keybind to reset P/T . CTRL + ALT + 0

**Comment by skwerlman** [comment]
i still think the default behavior should change here. in what scenario do you want to flip a card face down and _retain_ its P/T?
